dimod/include/dimod/quadratic_model.h

```diff
@@ -1094,6 +849,123 @@ class BinaryQuadraticModel : public QuadraticModelBase<Bias, Index> {
     Vartype vartype_;
 };
 
+template<class T>
+struct VarInfo {
+    Vartype vartype;
+    T lb;
+    T ub;
+
+    VarInfo(Vartype vartype, T lb, T ub) : vartype(vartype), lb(lb), ub(ub) {}
+};
+
+template <class Bias, class Index = int>
+class QuadraticModel : public QuadraticModelBase<Bias, Index> {
+ public:
+    /// The type of the base class.
+    using base_type = QuadraticModelBase<Bias, Index>;
+
+    /// The first template parameter (Bias).
```

Comment:
If the template parameter order is changed, these comments need to be changed. ie first, second.